jit/v1/jit_v1.2.0-Beta-2.yaml

```diff
@@ -0,0 +1,1312 @@
+          description: The identifying number of the consignee or shipper (Individual or entity) used for tax purposes.
+          example: CVR-25645774
+          maxLength: 20
+        taxReference2:
+          type: string
+          x-stoplight:
+            id: vm15r1xiq9m6h
+          description: Optional second identifying number of the consignee or shipper (Individual or entity) used for tax purposes.
+          example: CVR-25645774
+          maxLength: 20
+        publicKey:
+          type: string
+          x-stoplight:
+            id: 1q30laewgdk16
+          description: The public key used for a digital signature.
+          example: eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IkFzaW
+          maxLength: 500
+        address:
+          $ref: '#/components/schemas/Address'
+          x-stoplight:
+            id: x2gzbq5fkm5ft
+          description: An object for storing address related information.
+        identifyingCodes:
+          x-stoplight:
+            id: l8vzjedgfblkv
+          type: array
+          items:
+            $ref: '#/components/schemas/IdentifyingCode'
+            x-stoplight:
+              id: jk5yh30tlpt07
+        nmftaCode:
+          type: string
+          x-stoplight:
+            id: ckn2a425zsu31
+          example: MMCU
+          maxLength: 4
+          deprecated: true
+          description: "The Standard Carrier Alpha Code (SCAC) provided by NMFTA.\r\n\r\n**Deprecated**: Replaced by DCSAResponsibleAgencyCode and partyCode defined in the identifyingCodes list."
+    OperationsEvent:
+      title: OperationsEvent
+      x-stoplight:
+        id: er6q4hmrbkx7g
+      type: object
+      description: The operations event entity is a specialization of the event entity to support specification of data that only applies to an operations even
+      required:
+        - eventCreatedDateTime
+        - eventType
+        - eventClassifierCode
+        - eventDateTime
+        - operationsEventTypeCode
+        - publisher
+        - publisherRole
+        - transportCall
+      properties:
+        eventID:
+          type: string
+          x-stoplight:
+            id: os6w8s25806sn
+          description: "The unique identifier for the event (the message - not the source).\r\n\r\n**NB**: This field should be considered Metadata."
+          format: uuid
+          example: 3cecb101-7a1a-43a4-9d62-e88a131651e2
+        eventCreatedDateTime:
+          type: string
+          x-stoplight:
+            id: 8xt12fvmoauh0
+          description: "The timestamp of when the event was created.\r\n\r\n**NB**: This field should be considered Metadata"
+          format: date-time
+          example: '2021-01-09T14:12:56+01:00'
+        eventType:
+          type: string
+          x-stoplight:
+            id: ebjnq1uozwmym
+          description: "The Event Type of the object - to be used as a discriminator.\r\n\r\n**NB**: This field should be considered Metadata."
+          example: OPERATIONS
```

Comment:
Must be an enum

Reply:
<img width="284" alt="image" src="https://github.com/dcsaorg/DCSA-OpenAPI/assets/138600841/12c9f6c7-222d-4e41-b171-7268f9208468">

Should I include OPERATIONS as well?

Reply:
It should **only** include `OPERATIONS`
<img width="518" alt="image" src="https://github.com/dcsaorg/DCSA-OpenAPI/assets/3489388/85cbf8ae-28ad-47d1-b876-2c073dd82d94">